### PR TITLE
Update the examples to use the new way of launching services

### PR DESCRIPTION
**Pull request by pbirkhol-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_
- [x]  Automatically post PR comments with images for G code changes? _(Recommended for small changes)_

### What does this Pull Request accomplish?

The Keysight example uses the old way of launching measurements, which means that when running the measurement as an exe, the front panel is always displayed. This change makes the Keysight example match the other examples.

It also updates the build spec settings of a few other examples that had older settings that caused glitching when opening an exe.

### What testing has been done?

Built and ran the example and made sure that the front panel didn't show.

### CODE REVIEW

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj [hunk 1]
```diff
@@ -128,6 +156,7 @@
 				<Property Name="Source[8].sourceInclusion" Type="Str">TopLevel</Property>
 				<Property Name="Source[8].type" Type="Str">VI</Property>
 				<Property Name="SourceCount" Type="Int">9</Property>
+				<Property Name="TgtF_fastFileFormat" Type="Bool">true</Property>
```
Comment (pbirkhol-ni) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj:159]: This is the only change I made to the project file. Everything else is LabVIEW's doing.

**Review by jasonmreding — APPROVED** [review]

File: Source/Example Measurements/Game Of Life/Game Of Life.lvproj
Comment (pbirkhol-ni) [inline Source/Example Measurements/Game Of Life/Game Of Life.lvproj:1]: In this project, the only setting I changed was the window runtime position of the Run Service VI.

File: Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj
Comment (pbirkhol-ni) [inline Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj:1]: In this project, the I changed the window runtime position of the Run Service VI and I enabled Fast File Format.

File: Source/Example Measurements/NI-DCPower Source DC Voltage/NIDCPowerSourceDCVoltage.lvproj
Comment (pbirkhol-ni) [inline Source/Example Measurements/NI-DCPower Source DC Voltage/NIDCPowerSourceDCVoltage.lvproj:1]: In this project, the I changed the window runtime position of the Run Service VI and I enabled Fast File Format.

**Review by dixonjoel — APPROVED** [review]

**Pull request merged**